app/src/main/java/com/keylesspalace/tusky/components/account/AccountActivity.kt

```diff
@@ -135,6 +138,8 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidI
 
     private lateinit var adapter: AccountPagerAdapter
 
+    private val dateJoinedFormatter by lazy { SimpleDateFormat("MMM, yyyy", Locale.getDefault()) }
```

Comment:
Why lazy?

Reply:
Habit. As far as I remember, constructing SimpleDateFormats is an expensive operation, so it made sense to delay its construction until it was needed (if needed).

Reply:
I don't think it is a problem to immediately initialize it. If you want to delay the construction you could also move it into the `updateAccountJoinedDate` method, it is not called that often

Reply:
Done, since it's not often called, then we can just locally create it and call it a day. Updated.